src/ModixTranslator/HostedServices/TranslatorHostedService.cs

```diff
@@ -53,24 +53,33 @@ public TranslatorHostedService(ILogger<TranslatorHostedService> logger, ITransla
 
             var fromLangName = $"{safeLang}-to-{safeGuildLang}";
             var toLangName = $"{safeGuildLang}-to-{safeLang}";
-            var fromLangTopic = await _translation.GetTranslation(guildLang, lang, $"Responses will be translated to {guildLang} and posted in this channel's pair `#{fromLangName}`");
-            var toLangTopic = $"Responses will be translated to {lang} and posted in this channel's pair `#{toLangName}`";
 
             var fromLangChannel = await guild.CreateTextChannelAsync(fromLangName, p =>
             {
                 p.CategoryId = category.Id;
-                p.Topic = fromLangTopic;
             });
 
-            var ToLangChannel = await guild.CreateTextChannelAsync(toLangName, p =>
+            var toLangChannel = await guild.CreateTextChannelAsync(toLangName, p =>
             {
                 p.CategoryId = category.Id;
-                p.Topic = toLangTopic;
             });
+
+            await fromLangChannel.ModifyAsync(async p =>

```

Comment:
Why is this modifying the channel instead of just creating it with the topic?

Reply:
To mention a channel, you need the id; a channel mention is actually just a `<#{id}>` - thus, the channel must be created first to be able to mention it. I suppose I could create one, then create the other with the mention straight away, but that would be a little inconsistent imo.